es-ds-docs/pages/organisms/cta-card.vue

```diff
@@ -0,0 +1,211 @@
+                </p>
+            </b-col>
+        </b-row>
+
+        <div class="mb-500">
+            <h2 class="mb-200">Default with zip code form</h2>
+            <b-row class="justify-content-center justify-content-lg-start">
+                <b-col
+                    sm="8"
+                    md="6"
+                    lg="4"
+                    xxl="3">
+                    <es-cta-card>
+                        <template #heading> Easily find what solar costs in your area </template>
+                        <template #image>
+                            <nuxt-img
+                                alt="Map showing quotes in your area"
+                                class="w-100"
+                                height="257px"
+                                src="https://a-us.storyblok.com/f/1006159/220x257/18c519c201/mfdp-callout.webp/m/filters:quality(100)"
+                                width="220px" />
+                        </template>
+                        <template #cta>
+                            <es-zip-code-form
+                                input-id="default-with-zip-form"
+                                privacy-policy-link="https://www.energysage.com/privacy-policy/"
+                                url="https://www.energysage.com/market/start/">
+                                <template #buttonText> See local offers </template>
+                            </es-zip-code-form>
+                        </template>
+                    </es-cta-card>
+                </b-col>
+            </b-row>
+        </div>
+
+        <div class="mb-500">
+            <h2 class="mb-200">Image first with button</h2>
+            <b-row class="justify-content-center justify-content-lg-start">
+                <b-col
+                    sm="8"
+                    md="6"
+                    lg="4"
+                    xl="5"
+                    xxl="4">
+                    <es-cta-card image-first>
+                        <template #image>
+                            <nuxt-img
+                                alt="Heat pumps"
+                                class="w-100"
+                                height="150"
+                                src="https://a-us.storyblok.com/f/1006156/159x150/af4d52baaa/heat-pump-icon.svg"
+                                width="150" />
+                        </template>
+                        <template #heading> Upgrade your heating and cooling with a new heat pump </template>
+                        <template #subtitle>
+                            <p>100% free to use, 100% online</p>
+                            <p>Access the lowest prices</p>
+                            <p>Free, expert advice along the way</p>
+                        </template>
+                        <template #cta>
+                            <es-button
+                                class="w-100"
+                                href="https://heatpumps.energysage.com/welcome">
```

Comment:
Not that an example on a docs page matters, but the button in the example "Image first with button" no longer goes to the heat pumps page. It worked in the [old version](https://design.energysage.dev/organisms/es-cta-card/) of the CTA card. That [markup in the docs page](https://github.com/EnergySage/es-ds/blob/main/es-design-system/pages/organisms/es-cta-card.vue#L94) hasn't changed.

I searched the v3 codebase for examples of buttons that use an `href` parameter (as opposed to `:href`) and I didn't find any. There are two examples in the v2 Design System, [EsCtaBanner](https://design.energysage.dev/organisms/es-cta-banner/) and [EsCtaCard](https://design.energysage.dev/organisms/es-cta-card).

I feel like I'm missing something here.

Is it because [PrimeVue's Button](https://v3.primevue.org/button/) doesn't accept an `href` prop while the [BootstrapVue Button](https://bootstrap-vue.org/docs/components/button#element-type) does? I'm wondering if that was a side effect and we need to be more explicit about `hrefs` on buttons.

Reply:
Good catch! Yeah our (so far) implementation of EsButton doesn't support the `href` prop, and we got it for free before because Bootstrap Vue's button did. We have [a ticket](https://energysage.atlassian.net/browse/CED-1863) to fix that, and I've updated its acceptance criteria to verify that it works on this docs page as well.